cloud.profiles.d/edx_sandbox.conf

```diff
@@ -0,0 +1,31 @@
+# This profile is for instances launched for the purpose of
+# building edX sandbox AMIs.
+
+edx_sandbox_ami:
+  provider: mitx
+  size: m3.medium
+  image: ami-427a392a
+  ssh_username: ubuntu
+  ssh_interface: public_ips
+  securitygroupid: sg-1d437d65
+  subnetid: subnet-8671faac
+  block_device_mapping:
+    - DeviceName: /dev/xvda
+      Ebs.VolumeSize: 50
+      Ebs.VolumeType: gp2
+  ebs_optimized: False
```

Comment:
I think that M series instances include EBS optimization, so it might make sense to set this to True. Please verify with AWS.